_includes/pbc/all/install-features/202304.0/install-the-gift-cards-feature.md

```diff
@@ -0,0 +1,1338 @@
+
+
+This document describes how to integrate the [Gift Cards feature](/docs/pbc/all/gift-cards/{{site.version}}/gift-cards.html) into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Gift Cards feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME   | VERSION | INTEGRATE GUIDE |
```

Comment:
I added links to IGs. Make sure they are correct.

Reply:
@ilyakubanov I left all links to other IGs with old version 202212.0 confirm pls that it's right

Reply:
@oleksandr-stoilovskyi you option is not correct. I replaced versions with {{site.version}}